doc/zsdoc/zinit-side.zsh.adoc

```diff
@@ -138,12 +138,12 @@ Uses feature(s): _trap_
 Called by:
 
  zinit-autoload.zsh/.zinit-run-delete-hooks
- zinit-install.zsh/∞zinit-atclone-hook
- zinit-install.zsh/∞zinit-atpull-e-hook
- zinit-install.zsh/∞zinit-atpull-hook
- zinit-install.zsh/∞zinit-make-ee-hook
- zinit-install.zsh/∞zinit-make-e-hook
- zinit-install.zsh/∞zinit-make-hook
+ zinit-install.zsh/$'\342'$'\210'$'\236'zinit-atclone-hook
```

Comment:
Odd. That shouldn't have happened.

Reply:
hmm could you provide Dokerfile for `make doc` environment?

I use ubuntu:latest image and only install `git make zsh tree` through apt